<ui version="4.0" >
 <class>PhonemeEditorWidget</class>
 <widget class="QWidget" name="PhonemeEditorWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>133</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="autoFillBackground" >
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="_4" >
   <property name="leftMargin" >
    <number>1</number>
   </property>
   <property name="topMargin" >
    <number>1</number>
   </property>
   <property name="rightMargin" >
    <number>1</number>
   </property>
   <property name="bottomMargin" >
    <number>6</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>95</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable" >
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="m_phonemeScrollArea" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>490</width>
        <height>95</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <widget class="TimeLineFrame" name="m_phonemes" >
       <property name="geometry" >
        <rect>
         <x>70</x>
         <y>40</y>
         <width>390</width>
         <height>31</height>
        </rect>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>31</height>
        </size>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>185</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>185</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>185</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip" >
        <string>Here you can edit the phonemes.</string>
       </property>
       <property name="statusTip" >
        <string>Here you can edit the phonemes.</string>
       </property>
       <property name="whatsThis" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Here you can edit the phonemes.&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">You can change their timing and their value.&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="autoFillBackground" >
        <bool>true</bool>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
      <widget class="QLabel" name="phonemeLabel" >
       <property name="geometry" >
        <rect>
         <x>1</x>
         <y>47</y>
         <width>64</width>
         <height>17</height>
        </rect>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Phonemes:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="textLabel" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="geometry" >
        <rect>
         <x>1</x>
         <y>12</y>
         <width>64</width>
         <height>17</height>
        </rect>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Text:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QFrame" name="m_text" >
       <property name="geometry" >
        <rect>
         <x>70</x>
         <y>5</y>
         <width>390</width>
         <height>31</height>
        </rect>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>31</height>
        </size>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>185</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>185</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>185</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip" >
        <string>Here you can edit the phoneme text.</string>
       </property>
       <property name="statusTip" >
        <string>Here you can edit the phoneme text.</string>
       </property>
       <property name="whatsThis" >
        <string>Here you can edit the phoneme text.
It is not important that the words are written correctly, it is more important that the spelling is obvious.</string>
       </property>
       <property name="autoFillBackground" >
        <bool>true</bool>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth" >
        <number>1</number>
       </property>
       <property name="midLineWidth" >
        <number>0</number>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="sizeConstraint" >
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QToolButton" name="m_playButton" >
       <property name="text" >
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_stopButton" >
       <property name="text" >
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_saveButton" >
       <property name="text" >
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_closeButton" >
       <property name="text" >
        <string>Sound Editor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeLineFrame</class>
   <extends>QFrame</extends>
   <header>timelineframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
